### PR TITLE
#170764565 setup sequelize

### PR DESCRIPTION
**Pull request by Elvis-rugamba**
#### What does this PR do?
setup sequelize
#### Description of task to be completed

- Install sequelize-cli, pg-hstore, sequelize dependencies
- Create .sequelizerc file and modify path to migrations, models, seeders and config folder
- Create dummy user model & migration file to test the configuration if it works
- Create user seed
- Modify migration and model file to enforce unique and required fields
- Add babel-plugin-add-module-exports(It transforms 'export default' to 'exports.default') dev dependency to refactor generated files from es5 to es6
- Add required setup to run database migration in the README

#### How can this be manually tested?
After cloning repo, Cd into it,

- RUN `npm install`,

- Open .env and edit the DATABASE_URL to match your respective database information

- RUN ` node_modules/.bin/sequelize db:migrate` to migrate a dummy user table

#### What are the relevant Pivotal Tracker Stories?
#170764565
#### Screenshots (if appropriate)
None

### CODE REVIEW

**Review by shaazk — APPROVED** [review]
looks good to me

**Review by Skemc — APPROVED** [review]
Looks good to me

File: package.json [hunk 1]
```diff
@@ -14,16 +19,17 @@
...
+    "cross-env": "^6.0.3",
     "eslint": "^6.8.0",
     "eslint-config-airbnb": "^18.0.1",
     "eslint-plugin-import": "^2.20.0",
+    "nodemon": "^1.19.4",
     "mocha": "^7.0.0",
-    "nyc": "^15.0.0",
-    "nodemon": "^1.19.4"
+    "nyc": "^15.0.0"
+    "coveralls": "^3.0.9"
```
Comment (ajimae) [inline package.json]: There is an error here, kindly fix this.
Thank you.

**Review by ajimae — CHANGES_REQUESTED** [review]
Please fix the error in this file

Thank you.

File: package.json (hunk 1 above)
Comment (Elvis-rugamba) [reply package.json]: Fixed

Comment (ajimae) [reply package.json]: I think the error is still there, please fix it

**Review by ajimae — APPROVED** [review]
This looks good to me!

**Pull request merged**